src/main/app/src/constants/ArticleCard.js

```diff
@@ -4,7 +4,7 @@ import { Button, Card, CardActionArea, CardActions, CardContent, Typography, Box
 import { Link } from 'react-router-dom';
 
 // This is a stateless, functional React component used to render each resource in a card format. 
-const ArticleCard = ({ data, updateActivity }) => {
+const ArticleCard = ({ data, onClickFunction, parameters, buttonText }) => {
   const classes = useStyles();
 
   if (!data.title) return <div />
```

Comment:
Indent lines 11-28 by two space.

Reply:
Why? I don't think we need to (https://google.github.io/styleguide/jsguide.html#formatting-block-indentation) but I could be wrong.

Reply:
Indentation isn't needed here. The return statement is part of the function body.

Reply:
Oops. I thought it was part of the if body. Sorry!